### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by shubambhasin**

### CODE REVIEW

**Comment by Gagan-Deogan** [comment]
Amazing work shubam, everything is working super fine. 
This is a list of some issues which you can improve on.
1. Structure your files in a way that easily maintainable like put all your image in one assets folder src--assets--Image
2. you have use the inline style, if you make a CSS class and use it to replace the inline style that would reduce work.
3. You are using h1 with classname='h1' instead you can style h1 element in CSS which again reduce your work.

keep the good work going @shubambhasin

**Comment by shubambhasin** [comment]
Will make the changes ! Thanks !

**Review by Ashish-j-shetty — COMMENTED** [review]
Everything looks good to me

**Comment by vineet-singhal2312** [comment]
code is awesome.....
remove all the comments so it will look more better!!

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,60 @@
...
+import TextUtils from "./components/library/TextUtils";
+import Sidebar from "./components/Sidebar";
+// import TopNav from "./components/TopNav";
+import "./ease/ease.css"  
+import "./style.css"; 
+
 
 function App() {
+
+  // const [theme, setTheme] = useState("dark")
```
Comment (suyashpradhan) [inline src/App.js:26]: Remove comments from the entire app.

**Review by suyashpradhan — COMMENTED** [review]
Everything is great, just a small suggestion if you are using the first letter as capital for one file then use it everywhere so that consistency will be maintained!

**Pull request merged**